.ci/magician/exec/runner.go

```diff
@@ -9,7 +11,24 @@ import (
 	cp "github.com/otiai10/copy"
 )
 
-type actualRunner struct{}
+type actualRunner struct {
+	dirStack *list.List
```

Comment:
suggestion... have runner initialize with a current directory
```
func NewRunner(directory) Runner {
```

Track the current directory in this struct 
```
	currentDirectory string(?)
	dirStack *list.List
```


when pushing just move current directory onto stack.

when popping pull from stack into current directory.

Reply:
Done